src/components/Item/MessageButton.js

```diff
@@ -77,6 +77,7 @@ const MessageButton = ({
         onClick={handleClickOpen}
         onMouseEnter={onMouseEnter}
         onMouseLeave={onMouseLeave}
+        disabled={currentUser.user._id === ownerId}
```

Comment:
I don't think there would ever be this case, because MessageButton is only ever rendered when the user is not the owner of the posting:

![image](https://user-images.githubusercontent.com/39810669/89251354-a74adc80-d5cb-11ea-8f92-90eacfc37976.png)

(It's fine to keep this code in though) xD

Reply:
oh weird I had it show up on mine on localhost. I guess keeping it doesnt hurt but ya good point. it is however good to have for double safe-checking

Reply:
oh shoot, I had a bug in the code sorry. Forgot to set "currentUser" as "state.currentUser". Is why it was showing up.

(but as you say, it's good to have your fallback)